README.md

```diff
@@ -530,6 +530,10 @@ On Windows it is suggested that MSI be used for installation.
 
 Please see [Extensions.md](Extensions.md) on how to build and run conditionally built dependencies and extensions.
 
+### Third parties
+
+Please see [ThirdParties.md](ThirdParties.md) on how MiNiFi builds and uses third party libraries and how you can add new ones.
```

Comment:
This seems more developer focused, perhaps this could be split into the contrib guide?

Reply:
Yep, that's fair, moved the reference to `CONTRIB.md`